ras_party/views/account_view.py

```diff
@@ -76,6 +76,12 @@ def resend_verification_email(party_uuid):
     return make_response(jsonify(response), 200)
 
 
+@account_view.route("/resend-account-email-change-notification/<party_uuid>", methods=["POST"])
+def resend_account_email_change_verification(party_uuid):
+    response = account_controller.resend_account_email_change_verification_email_by_uuid(party_uuid)
+    return make_response(jsonify(response), 200)
```

Comment:
This can just be `jsonify(response)`

Reply:
for consistency we should add the http code I believe